utils/utils.go

```diff
@@ -601,3 +602,26 @@ func ExtractSha256FromResponseBody(body []byte) (string, error) {
 func Pointer[K any](val K) *K {
 	return &val
 }
+
+func SetEnvWithResetCallback(key, value string) func() {
```

Comment:
We try to avoid swallowing errors at any cost. Lert's have this function return the errors, and not swallow them.